.dockerdev/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG RUBY_VERSION
-FROM ruby:$RUBY_VERSION-slim-buster
+FROM ruby:$RUBY_VERSION-bullseye
```

Comment:
why not the `slim` and the current stable Debian `bookworm` variant?

Reply:
Hey @tvdeyen, with regards to the slim image, I'm not sure. I borrowed this commit from @kennyadsl in #5523. I will try and test with the slim image later today as this could help keep our Docker setup lean. I presume there is a justification for this change that we can later include in the commit message to help reviewers. If not, we should change to the slim image as you suggested.

With regards to `bullseye` or `bookworm`, I would prefer to update one step at a time if possible, but I would be willing to jump to `bookworm` if you think it's worth while.


Thanks for the review. Hope you are well!

Reply:
> Hey @tvdeyen, with regards to the slim image, I'm not sure. I borrowed this commit from @kennyadsl in #5523. I will try and test with the slim image later today as this could help keep our Docker setup lean. I presume there is a justification for this change that we can later include in the commit message to help reviewers. If not, we should change to the slim image as you suggested.

We already used the slim variant for the buster version. There should not be an issue using it for bullseye as well. There might be one or two missing libs, but those can be added as needed.

> With regards to `bullseye` or `bookworm`, I would prefer to update one step at a time if possible, but I would be willing to jump to `bookworm` if you think it's worth while.

Since this is the dev container I think we can be less cautious here. Our dev containers all use the `slim-bookworm` image with Solidus and it works. We do not want to update it to the lesser outdated variant, but to the current stable in order to not have to maintain this very soon again.